ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -30,7 +30,7 @@
     private static final By START_VISIT = By.id("org.openmrs.module.coreapps.createVisit");
     private static final By CONFIRM = By.cssSelector("#quick-visit-creation-dialog .confirm");
     private static final By STARTED_AT = By.className("active-visit-started-at-message");
-    private static final By VISIT_NOTE = By.id("referenceapplication.realTime.simpleVisitNote");
+    private static final By VISIT_NOTE = By.linkText("Visit Note");
```

Comment:
was this all that needed to be fixed? doesn't `By.id("referenceapplication.realTime.simpleVisitNote");` exist?

Reply:
Yes! Besides passing with firefox instances of both true and false, By.id() could not be retrieved by chrome true instance and refactoring to retrieving By.linkText() made things work perfect on both chrome and firefox instances